spec/flipper/adapters/sequel_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'helper'
+require 'sequel'
+
+Sequel::Model.db =  Sequel.sqlite
+Sequel.extension :migration, :core_extensions
+
+require 'flipper/adapters/sequel'
+require 'generators/flipper/templates/sequel_migration'
+require 'flipper/spec/shared_adapter_specs'
+
+RSpec.describe Flipper::Adapters::Sequel do
+  subject do
+    described_class.new(feature_class: feature_class,
+                        gate_class: gate_class)
+  end
+
+	let(:feature_class) { Flipper::Adapters::Sequel::Feature }
+	let(:gate_class) { Flipper::Adapters::Sequel::Gate }
+
+  before(:each) do
+    CreateFlipperTablesSequel.new(Sequel::Model.db).up
+    feature_class.dataset = feature_class.dataset
```

Comment:
I am not that familiar with sequel. What is the purpose of this dataset assignment?